packages/cli/test/unit/commands/__snapshots__/help.test.ts.snap

```diff
@@ -361,7 +361,7 @@ exports[`help command > bisect help output snapshots > bisect help column width
 
 exports[`help command > certs help output snapshots > certs help column width 40 1`] = `
 "
-  ▲ vercel certs [command] [options]
+  ▲ vercel certs command [options]
```

Comment:
why are we missing square brackets here?

Reply:
Because the subcommand is required for `vercel certs` 😄